infocmdb/attribute_group.go

```diff
@@ -0,0 +1,145 @@
+package infocmdb
+
+import (
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	v2 "github.com/infonova/infocmdb-sdk-go/infocmdb/v2/infocmdb"
+	utilError "github.com/infonova/infocmdb-sdk-go/util/error"
+)
+
+var boolToString = map[bool]string{false: "0", true: "1"}
```

Comment:
I am not super happy with the naming, but considering its usage - especially that it should be kept short - I can't come up with anything good myself right now.

Reply:
Changed it to convertBoolToString.